util/regexp-assemble/regexp-933161.data

```diff
@@ -1,3 +1,19 @@
+##! This is a data file used to generate a regular expression for a CRS rule. 
```

Comment:
Looks good! Maybe prefix and suffix comment could be added.

Reply:
Missed that one, thanks. Fixed.